about.html

```diff
@@ -0,0 +1,141 @@
+                <li class="list-menu"><a href="#" class="a-link">News</a></li>
+                <li class="list-menu"><a href="#" class="a-link">CC Campaign</a></li>
+            </ul>
+            </div>
+
+    </div>
+    </header>
+
+    <section class="hero">
+        <div class="container intro">
+        <!-- <h1>"Hello! Africa"</h1> -->
+        <h2>AFRICA TECH SUMMIT 2022</h2>
+        <P class="description">Africa Tech Summit Nairobi connects tech leaders from the African ecosystem and international players under one roof. Network with key stakeholders including tech corporates, mobile operators, fintechs, DeFi & crypto ventures, investors, leading start-ups, regulators and industry stakeholders driving business and investment forward.</P>
+        
+        <P>Please contact us via Emial for further questions <br> concerning Africa Tech Summit 2022</P>
+
+        <p><span>africatechsummit2022@afsummit.org</span></p>
+        </div>
+    </section>
+
+    <section class="summit-logo">
+        <div class="summit-head">
+            <h2>Africa Tech<br> Summit 2022 Logo</h2>
+            <hr>
+        </div>
+        <p>The logo for Africa Tech Summit 2022 was decided through the logo competition from 8 June to 7 July</p>
+        <div class="afrilogo"><img src="./images/tech-logo.png" alt="Africa Tech summit logo"></div>
+    </section>
+
+    <section class="past-events">
+        <div class="past">
+        <h2>See Past Africa Tech Summits</h2>
+        <hr>
+        </div>
+        <div class="time"><p>Take a look at the last two Africa Tech summit that took place in Kenya and Rwanda</p></div>
+    
+        
+
+        <div class="event-container">
+            <div class="second-event">
+                <div class="event-content">
+                    <h2 class="year">2021</h2>
+                    <p class="city">Africa Tech Summit 2021 in Kigali</p>
+                </div>
+
+            </div>
+            <div class="first-event">
+                <div class="event-content">
+                    <h2 class="year">2020</h2>
+                    <p class="city"> Africa Tech Summit 2020 in Nairobi</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="partners">
+        <div class="brand">
+        <h3>Partners</h3>
+        <hr>
+        </div>
+        <div class="brand-logo">
+            <span class="partnerbrand_1">Facebook</span>
+            <span class="partnerbrand_2">Paystack</span>
+            <span class="partnerbrand_1">Uber</span>
+            <span class="partnerbrand_2">Google</span>
+            <span class="partnerbrand_1">Jumia</span>
+            <span class="partnerbrand_2">Twitter</span>
+        </div>
+
+    </section>
+
+    <footer>
+        <div class="base-container">
+        <img class="desk-logo" src="./images/trans.png" alt="Africa Tech Summit Logo">
+        <img class="mobile-logo" src="./images/newlogo.png" alt="Africa Tech Summit logo">
```

Comment:
Please add `a` tag to the logo in the about page footer. However, for the mobile version, please make sure that when users click on the footer's logo it directs them to the home page.

Reply:
I have added the link tags as suggested